scripts/redact_cli_py/redact/utils/pdf_renderer.py

```diff
@@ -2,53 +2,24 @@
 # Licensed under the MIT License. See License.txt in the project
 # root for license information.
 
-import sys
 from PIL import Image
-
-import ctypes
-import pypdfium as pdfium
-
-WHITE = 0xFFFFFFFF
+import pypdfium2 as pdfium
 
 
 class PdfRenderer:
+
     def __init__(self):
-        # Initiate PDFium - This only needs to happen once
-        pdfium.FPDF_InitLibraryWithConfig(pdfium.FPDF_LIBRARY_CONFIG(2, None, None, 0))
+        pass
 
     # Strong Assumption: Assume all PDFs are one page PDF
```

Comment:
Why only render the first page? You could render all pages of the PDF, or ask the user which pages to render...